app/Http/Controllers/PayloadController.php

```diff
@@ -100,8 +100,15 @@ public function update(Request $request, Payload $payload)
      * @param  \App\Models\Payload  $payload
      * @return \Illuminate\Http\Response
      */
-    public function destroy(Payload $payload)
+    public function destroy(Webhook $webhook, Payload $payload)
     {
-        //
+        try {
+            $this->payloadRepository->delete($payload->id);
+
+            return redirect()->route('webhooks.edit', $webhook)
+                             ->with('messageSuccess', 'This payload successfully deleted');
+        } catch (Exception $exception) {
+            return redirect()->back()->with('messageFail','Delete failed. Something went wrong');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 111</br>No space found after comma in function call